### PR TITLE
AUTO-724 - Add connections page

### PR DESCRIPTION
**Pull request by andrsvsrg**

### CODE REVIEW

File: src/automation.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ const routes = prepareRoutes({
   flowSlAGoal       : '/api/app/:appId/automation/flow/:flowId/version/:versionId/sla/goals/:id',
   SLACalendars      : '/api/app/:appId/automation/flow/sla/calendar',
   SLACalendar       : '/api/app/:appId/automation/flow/sla/calendar/:id',
+
+  flowVersionsByConnectionIds: '/api/app/:appId/automation/oauth/connections/flows/versions/by-connection-ids',
```
Comment (stanislav-grin) [inline src/automation.js:19]: Why not just `/api/app/:appId/automation/flows/versions/by-connection-ids`?

Comment (Valodya) [reply src/automation.js:19]: if the idea was to retrieve all connection usages I would add the following route

```
'/api/app/:appId/oauth2/connections/usages'
``` 
and on the server, we collect all information from all known places () and then it returns connections with all aggregated usages

File: src/oauth2-integration.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { prepareRoutes } from './utils/routes'
 
 const routes = prepareRoutes({
   connectionURL            : '/api/node-server/api/app/:appId/integration/connection/oauth/url',
+  connectionsUsages        : '/api/node-server/api/app/:appId/oauth/connections/usages',
```
Comment (Valodya) [inline src/oauth2-integration.js:5]: instead of `api` group it should be `manage`

Comment (Valodya) [reply src/oauth2-integration.js:5]: `/api/node-server/manage/app/:appId/integration/connections/usages`

**Pull request merged**